pkgs/development/libraries/wxwidgets/3.0/default.nix

```
@@ -7,8 +7,8 @@
 , libGLU, libGL
 , compat24 ? false, compat26 ? true, unicode ? true
 , withGtk2 ? true
-, withWebKit ? false, webkitgtk
-, AGL, Carbon, Cocoa, Kernel, QTKit
+, withWebKit ? !stdenv.isDarwin, webkitgtk  # Disable webkitgtk on darwin pending #126101
```

Actually, the original value was already false, so setting it to `!stdenv.isDarwin` would enable it on non-darwin. And I don't see any override in all-packages.nix
```suggestion
, withWebKit ? false, webkitgtk
```